engine/table/src/main/java/io/deephaven/engine/util/PyCallableWrapper.java

```diff
@@ -142,11 +157,10 @@ private void prepareSignature() {
                 throw new IllegalArgumentException(
                         "numba vectorized function must have an explicit signature: " + pyCallable);
             }
-            // numba allows a vectorized function to have multiple signatures, only the first one
-            // will be accepted by DH
+            // numba allows a vectorized function to have multiple signatures
             if (params.size() > 1) {
                 throw new UnsupportedOperationException(
-                        "Multiple signatures on numba vectorized functions aren't supported yet.");
+                        "Multiple signatures on numba vectorized functions aren't supported yet. " + pyCallable);
```

Comment:
Tell me what you're printing...
```suggestion
                        pyCallable + " has multiple signatures; this is not currently supported for numba vectorized functions");
```